metashare/api/jobs.py

```diff
@@ -71,6 +76,47 @@ def _create_branches_on_github(*, user, repo_id, project, task):
     return task_branch_name
 
 
+def alert_user_about_expiring_org(*, org, days):
+    from .models import User, ScratchOrg
+
+    # if scratch org is there
+    try:
+        org.refresh_from_db()
+        user = org.owner
```

Comment:
@wlonk I made this change to use the org owner instead of the creating user, anticipating eventually allowing transferring ownership of orgs.